addons/sale_timesheet/models/project.py

```diff
@@ -326,8 +326,10 @@ class ProjectTask(models.Model):
     def _get_default_partner_id(self, project, parent):
         res = super()._get_default_partner_id(project, parent)
         if not res and project:
-            if project.pricing_type == 'employee_rate':
-                return project.sale_line_employee_ids.sale_line_id.order_partner_id[:1]
+            # project in sudo if the current user is a portal user.
+            related_project = project if not self.user_has_groups('!base.group_user,base.group_portal') else project.sudo()
```

Comment:
so portal users have more access than normal users?

Reply:
The method is private, and the goal is to retrieve a default partner_id. Is it really an issue or am I missing something? :)